google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/MessageDispatcher.java

```diff
@@ -273,7 +273,8 @@ public void processReceivedMessages(List<com.google.pubsub.v1.ReceivedMessage> r
       ackHandlers.add(new AckHandler(pubsubMessage.getAckId(), messageSize));
     }
     Instant expiration = now.plus(messageDeadlineSeconds * 1000);
-    logger.log(Level.INFO, "Received " + responseMessages.size() + " messages at " + now);
+    logger.log(
+        Level.FINER, "Received {0} messages at {1}", new Object[] {responseMessages.size(), now});
```

Comment:
Why are you explicitly allocating an object array? This is a varargs parameter, right? 

`logger.log(Level.FINER, "Received {0} messages at {1}", responseMessages.size(), now);`

Also, I think it's most appropriate to use `FINE` for operation-level messages that are summary-like - here, we are just giving message counts.